### ISSUE TITLE
[BUG 🪲] When the window is not maximized, the obtained image is incorrect.

### ISSUE THREAD

**Issue by enximi** [NiiightmareXD/windows-capture#16]
**Describe the bug 🐛**
When the window is not maximized, the obtained image is incorrect.

**Expected behavior 📝**
Correctly capture windows without maximizing them.

**OS 🤖**
 - Version: [Windows 11]
 - Build[23H2]

**Additional context ➕**
This is an image captured using windows-capture when the control panel window is not maximized.
![img1](https://github.com/NiiightmareXD/windows-capture/assets/70036307/78a632aa-8a02-48a4-a85e-eacc5c79886d)
This is the correct image captured using other screenshot tool.
![img2](https://github.com/NiiightmareXD/windows-capture/assets/70036307/074d90fe-fef1-44a1-ae5f-3627d4c73a8a)
If the window is maximized, then windows-capture can capture the correct image.
This is an image obtained when the control panel window is maximized.
![img3](https://github.com/NiiightmareXD/windows-capture/assets/70036307/bd765ae3-9ff7-4c2d-b9a6-b46917b03838)

My code is as follows. Compared with the example, I only modified the window to be captured. I have also run it using the foreground window. I ran it in RustRover﻿, so the captured window is RustRover's window﻿. When the RustRover ﻿window is not maximized, the obtained image will also be distorted like what was described above.

```rust
use windows::Win32::Foundation::HWND;
use windows_capture::{
    capture::WindowsCaptureHandler,
    frame::{Frame, ImageFormat},
    graphics_capture_api::InternalCaptureControl,
    settings::{ColorFormat, Settings},
    window::Window,
};

struct Capture;

impl WindowsCaptureHandler for Capture {
    type Flags = String;

    type Error = Box<dyn std::error::Error + Send + Sync>;

    fn new(message: Self::Flags) -> Result<Self, Self::Error> {
        println!("Got The Flag: {message}");

        Ok(Self {})
    }

    fn on_frame_arrived(
        &mut self,
        frame: &mut Frame,
        capture_control: InternalCaptureControl,
    ) -> Result<(), Self::Error> {
        println!("New Frame Arrived");

        frame.save_as_image("image3.png", ImageFormat::Png)?;

        capture_control.stop();

        Ok(())
    }

    fn on_closed(&mut self) -> Result<(), Self::Error> {
        println!("Capture Session Closed");

        Ok(())
    }
}

fn main() {
    let my_window = Window::from_raw_hwnd(HWND(0x000A0B8E));

    let settings = Settings::new(
        my_window,
        Some(true),
        None,
        ColorFormat::Rgba8,
        "It Works".to_string(),
    )
    .unwrap();

    Capture::start(settings).expect("Screen Capture Failed");
}
```

**Comment by NiiightmareXD** [comment]
Thanks for reporting the bug fixed in v1.0.60.

**Comment by NiiightmareXD** [comment]
Please notify me if it's still not fixed but I tested v1.0.60 in control panel and its fixed 😊.

**Comment by enximi** [comment]
Fixed. Thanks.
